src/System.Data.SqlClient/tests/FunctionalTests/SqlConnectionBasicTests.cs

```diff
@@ -135,6 +135,29 @@ public void ConnectionTimeoutTestWithThread()
             Console.WriteLine($"ConnectionTimeoutTestWithThread: Elapsed Time {theMax} and threshold {threshold}");
         }
 
+        [Fact]
+        public void ExceptionsWithMinPoolSizeCanBeHandled()
+        {
+            int count = 0;
+            int expected = 3;
+            string connectionString = $"Data Source={Guid.NewGuid().ToString()};uid=random;pwd=asd;Connect Timeout=2; Min Pool Size=3";
```

Comment:
Does this Connect Timeout mean this will take 2 seconds to fail?  And we do it 3 times?  Might want to make the test outerloop then.

Reply:
The test will try for a max of 2 seconds. However it could fail earlier. But it will go upto 2 seconds.
Outerloop makes sense. Thanks.